pkg/apis/apps/v1alpha1/generated.proto

```diff
@@ -24,6 +24,7 @@ package k8s.io.kubernetes.pkg.apis.apps.v1alpha1;
 import "k8s.io/kubernetes/pkg/api/resource/generated.proto";
 import "k8s.io/kubernetes/pkg/api/unversioned/generated.proto";
 import "k8s.io/kubernetes/pkg/api/v1/generated.proto";
+import "k8s.io/kubernetes/pkg/apis/policy/v1alpha1/generated.proto";
```

Comment:
I think you've somehow picked up this spurious line. If you remove it and rerun update-codegen.sh, does the proto generator put it back? @smarterclayton might have more insight.